### ISSUE TITLE
[BUG] Dropdown - Keyboard accessibility - When the user hit on up and down arrow keys on the keyboard, dropdown list items is not selected

### ISSUE THREAD

**Issue by HarshithaNous** [trimble-oss/modus-web-components#1194]
[BUG] Dropdown Web component - Keyboard accessibility - When the user hit on up and down arrow keys on the keyboard, dropdown list items is not selected

Steps to reproduce the behavior on Docs section:

1. Go to Dropdown web component
2. Click on Docs section
3. Scroll down to Dropdown Element
4. Hit on <kbd>Tab</kbd> key on the keyboard
5. Dropdown element should be selected
6. Hit on <kbd>Enter</kbd> key on the keyboard
7. Dropdown list should be displayed
8. Hit on <kbd>Up</kbd> and <kbd>Down</kbd> arrow keys to select the drop down list items
9. Dropdown list items is not selected accordingly

Steps to reproduce the behavior on Canvas section:

1. Go to Dropdown web component
2. Click on Canvas section
3. Hit on <kbd>Tab</kbd> key on the keyboard
4. Dropdown element should be selected
5. Hit on <kbd>Enter</kbd> key on the keyboard
6. Dropdown list should be displayed
7. Hit on <kbd>Up</kbd> and <kbd>Down</kbd> arrow keys to select the drop down list items
8. Dropdown list items is not selected accordingly

**Expected behavior**
The first item in the dropdown list should be preselected and when user hit on <kbd>up</kbd> and <kbd>down</kbd> arrow keys on the keyboard, the list of items on the dropdown should be selected accordingly

**Current behavior**
The list of items is not selected when the user hit on <kbd>up</kbd> and <kbd>down</kbd> arrow keys on the keyboard

Please refer the below screenshots:

![image](https://user-images.githubusercontent.com/129380495/232822339-f570782f-f83f-423a-9e08-f5df747d17dc.png)

![image](https://user-images.githubusercontent.com/129380495/232821811-645f697b-f307-4cc3-8ad7-f36ed6c69096.png)

- Browser Tested - Edge, Chrome and Firefox

**Comment by prashanth-offcl** [comment]
@HarshithaNous closing this issue based on @coliff comment. Kindly reopen this issue if you need more changes.
